docs/Basic-Guide.md

```diff
@@ -27,21 +25,36 @@ for more information.
        Equivalent or .NET 4.x Equivalent)**
 6. Go to **File** > **Save Project**
 
+## Setting up the Inference Engine
+
+In order to run pre-trained models of agents behaviors, you will need to set-up the 
+[Inference Engine](Inference-Engine.md). The Inference Engine is a general API to
+run Neural Network models in Unity that leverages existing inference libraries such 
+as TensorFlowSharp and CoreML. The models we ship with our examples and the models
+generated with our trainers use the TensorFlowSharp inference backend : 
```

Comment:
The models we ship with our examples and the models generated with our trainers use the TensorFlowSharp inference backend : They have a `.tf` extension. This means that you will not be able to use them
without first installing the TensorFlowSharp backend.

-->

Since the ML-Agents Toolkit uses TensorFlow for training neural network models, the output model format is TensorFlow and the model files include a `.tf` extension. Consequently, you need to install the TensorFlowSharp backend to be able to run these models within the Unity Editor.